nat-lab/tests/test_reconnections.py

```diff
@@ -109,17 +110,17 @@ async def test_mesh_reconnect(
             await ping(beta_connection, alpha.ip_addresses[0], 15)
 
         await client_alpha.simple_start()
-        await client_alpha.set_meshmap(api.get_meshmap(alpha.id))
 
-        await asyncio.gather(
-            client_alpha.wait_for_event_peer(beta.public_key, [telio.State.Connected]),
-            client_alpha.wait_for_event_on_any_derp([telio.State.Connected]),
-        )
-
-        await asyncio.gather(
-            client_alpha.wait_for_state_peer(beta.public_key, [telio.State.Connected]),
-            client_beta.wait_for_state_peer(alpha.public_key, [telio.State.Connected]),
-        )
```

Comment:
nit: Would still be nice to check if state is reported correctly on both sides after event is generated, but up to you

Reply:
didn't see your comment, this is useless :D

Reply:
generally `beta` is likely will not generate any events. So this is why I thought that asserting on them may be not the wisest thing

Reply:
but `.wait_for_state_peer` is safe probably, so I will readd it :D (changed my mind)